[["/home/kali/.local/lib/python3.11/site-packages/exceptiongroup/__init__.py",[false,[["__all__",true,13,null],["catch",true,2,["catch","/home/kali/.local/lib/python3.11/site-packages/exceptiongroup/_catch.py",12]],["format_exception",true,2,["format_exception","/home/kali/.vscode/extensions/ms-python.vscode-pylance-2023.12.1/dist/typeshed-fallback/stdlib/traceback.pyi",12]],["format_exception_only",true,2,["format_exception_only","/home/kali/.vscode/extensions/ms-python.vscode-pylance-2023.12.1/dist/typeshed-fallback/stdlib/traceback.pyi",12]],["print_exc",true,2,["print_exc","/home/kali/.vscode/extensions/ms-python.vscode-pylance-2023.12.1/dist/typeshed-fallback/stdlib/traceback.pyi",12]],["print_exception",true,2,["print_exception","/home/kali/.vscode/extensions/ms-python.vscode-pylance-2023.12.1/dist/typeshed-fallback/stdlib/traceback.pyi",12]],["BaseExceptionGroup",true,13,null],["ExceptionGroup",true,13,null]]]]]